modules/api-svc/package.json

```diff
@@ -63,6 +60,7 @@
   },
   "dependencies": {
     "@koa/cors": "^3.4.1",
+    "@mojaloop/api-snippets": "^14.2.3",
```

Comment:
I thought this was a Dev dependency, can you give me some insight as to why its not?

Reply:
that's because of this change:
https://github.com/mojaloop/sdk-scheme-adapter/blob/51f09c4ac942e7c701c6df9be4a6dff0444f893a/modules/api-svc/src/OutboundServer/index.js#L93